tsweb/log.go

```diff
@@ -45,6 +45,10 @@ type AccessLogRecord struct {
 	Bytes int `json:"bytes,omitempty"`
 	// Error encountered during request processing.
 	Err string `json:"err,omitempty"`
+	// The unique ID for this request. When a request fails due to an error, the
+	// ID is generated and displayed as part of the error message for the
+	// client.
```

Comment:
Any guidance here or elsewhere on its format? I guess more than digits. Whitespace? What's the allowed alphabet of characters?

Reply:
I didn't want `tsweb` to be particularly opinionated. Anything printable is fine. But I took your suggestion regarding adding a type and documenting the format there.